update/v1alpha1/types_update_status.go

```diff
@@ -0,0 +1,394 @@
+package v1alpha1
+
+import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+// +genclient
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// UpdateStatus reports status for in-progress cluster version updates
+//
+// Compatibility level 4: No compatibility is provided, the API can change at any point for any reason. These capabilities should not be used by applications needing long term support.
+// +openshift:compatibility-gen:level=4
+// +kubebuilder:object:root=true
+// +kubebuilder:subresource:status
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +kubebuilder:resource:path=updatestatuses,scope=Cluster
+// +openshift:api-approved.openshift.io=https://github.com/openshift/api/pull/2012
+// +openshift:file-pattern=cvoRunLevel=0000_00,operatorName=cluster-version-operator,operatorOrdering=02
+// +openshift:enable:FeatureGate=UpgradeStatus
```

Comment:
Does this gate already exist in the features package?

Reply:
Yes: https://github.com/openshift/api/blob/56ca5bb59cc11fe157839e7694c8093cc2e0f417/features/features.go#L452-L458